i18n/de.json

```
@@ -9,24 +9,39 @@
         "appVersion": "App-Version",
         "architecture": "Architektur",
         "batteryLevel": "Batteriestand",
+        "browserId": "Eindeutige ID für den Browser",
+        "campaignCreative": "Kampagneninhalt/-Creative (utm_content)",
+        "campaignCreativePosition": "Position des Kampagnen-Creatives",
+        "campaignMedium": "Kampagnenmittel (utm_medium)",
+        "campaignName": "Kampagnenname/-ID (utm_campaign)",
+        "campaignSource": "Kampagnenquelle (utm_source)",
+        "campaignTerm": "Kampagnensuchbegriff (utm_term)",
         "carrier": "Mobilfunkanbieter",
+        "consentState": "Einwilligungsstatus",
         "country": "Land",
+        "currency": "Währung",
         "deviceId": "Eindeutige ID für das Gerät",
         "developerScopedId": "Entwickler_innen-spezifische Geräte-ID (IDFV/ASID/ANDROID_ID)",
         "deviceName": "Name des Geräts",
         "diskFree": "Freier Speicherplatz",
         "diskTotal": "Gesamter Speicherplatz",
         "diskUsed": "Genutzter Speicherplatz",
+        "errorInformation": "Error-Informationen",
         "hashedAdvertisingId": "Gehashte Werbe-ID des Geräts",
         "installationId": "Eindeutige ID für die Installation",
         "installTime": "Zeitpunkt der Installation der App",
+        "interactedElement": "Element, mit dem die Nutzer_in interagiert hat",
         "isCharging": "Ladestatus",
+        "isConversion": "Ist dies ein Conversion-Event?",
         "isEmulator": "Ist das Gerät ein Emulator?",
         "isFirstLaunch": "Ist dies der erste Start?",
```

```suggestion
        "isFirstLaunch": "Ist dies der erste Start/Besuch?",
```
Analogous to the English translation